server.js

```diff
@@ -15,6 +15,15 @@ app.use(cors());
 
 app.set('port', process.env.PORT || 3001);
 
+app.get('/api/v1/races', async (request, response) => {
+  try {
+    const races = await database('races').select();
+    response.status(200).json(races);
+  }
+  catch(error) {
+    response.status(500).json({ error });
```

Comment:
3. Catch error if any data can be taken from DB and return error as response with status code `500`